app/com/linkedin/drelephant/exceptions/core/ExceptionFingerprintingRunner.java

```diff
@@ -223,7 +223,7 @@ private String parseExceptions(List<ExceptionInfo> exceptionInfoList, boolean is
     try {
       exceptionInJson = Obj.writeValueAsString(exceptionInfoList.subList(0,
           Math.min(exceptionInfoList.size(), NUMBER_OF_EXCEPTION_TO_PUT_IN_DB.getValue())));
-      logger.info(" Final Size of the exception List , which will be stored in db " + exceptionInJson);
+      logger.debug(" Final Size of the exception List , which will be stored in db " + exceptionInJson);
```

Comment:
If possible kindly raise another PR for this change, since it's a small change we can merge that PR in no time.